function-images/tests/chained-function-serving/consumer/consumer.go

```diff
@@ -34,8 +34,44 @@ import (
 	"google.golang.org/grpc"
 
 	pb "tests/chained-functions-serving/proto"
+
+	"go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/exporters/trace/zipkin"
+	"go.opentelemetry.io/otel/propagation"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
 )
 
+var logger = log.New(os.Stderr, "zipkin-example", log.Ldate|log.Ltime|log.Llongfile)
+
+// initTracer creates a new trace provider instance and registers it as global trace provider.
+func initTracer(url string) func() {
+	// Create Zipkin Exporter and install it as a global tracer.
+	//
+	// For demoing purposes, always sample. In a production application, you should
+	// configure the sampler to a trace.ParentBased(trace.TraceIDRatioBased) set at the desired
+	// ratio.
+	exporter, err := zipkin.NewRawExporter(
+		url,
+		zipkin.WithLogger(logger),
+		zipkin.WithSDKOptions(sdktrace.WithSampler(sdktrace.AlwaysSample())),
```

Comment:
sampling rate needs to be a parameter, so does Zipkin initialization... no changes needed right now but we need to keep it in mind. 

please keep this comment unresolved for now